functions/idvtoauth0/authzero.py

```diff
@@ -82,7 +85,10 @@ def update_user(self, user_id, new_profile):
         # This validates the JSON as well
         payload_json = json.dumps(payload)
 
-        self.conn.request("PATCH", "/api/v2/users/{}".format(user_id), payload_json, self._authorize(self.default_headers))
+        self.conn.request("PATCH",
```

Comment:
Let's stick with either double quotes or single. Other parts of this file have a combination of both. Personally I would go with single.